src/ApiProblem.php

```diff
@@ -239,7 +239,7 @@ public function setDetailIncludesStackTrace($flag)
      *
      * @return string
      */
-    protected function getDetail()
+    public function getDetail()
```

Comment:
The visibility on this should not change; access to processed detail is available via direct property access already (`$problem->detail`). (I'll revert on merge)